src/datascience-ui/variable-view/variableViewPanel.tsx

```diff
@@ -48,7 +51,6 @@ export class VariableViewPanel extends React.Component<IVariableViewPanelProps>
         // we can size and host it differently from the variable panel in the interactive window or native editor
         return (
             <div id="variable-view-main-panel" role="Main" style={dynamicFont}>
-                <button onClick={this.props.toggleVariableExplorer}>OPEN</button>
```

Comment:
Was just using this for testing and hadn't gotten around to removing it until now.

Reply:
Don't we need to localize the text `OPEN`? Can't remember what we do with such text in our webviews.

Reply:
@DonJayamanne I don't think we actually want to keep that button so no need to localize here.